python/agents/utils/objects.py

```
@@ -0,0 +1,228 @@
+    pagerDutyNotificationEnabled: Optional[bool] = None
+    reviewStatus: Optional[str] = None  # deployed, draft, etc.
+    approved: Optional[bool] = None
+    clobRewards: Optional[list[ClobReward]] = None
+    rewardsMinSize: Optional[int] = (
+        None  # would make sense to allow float but we'll see
+    )
+    rewardsMaxSpread: Optional[float] = None
+    spread: Optional[float] = None
+
+
+class ComplexMarket(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)